src/OpenSage.Game/Scene3D.cs

```diff
@@ -188,10 +188,6 @@ internal Scene3D(Game game, MapFile mapFile)
                                 position.Z += heightMap.GetHeight(position.X, position.Y);
 
                                 var gameObject = GameObject.FromMapObject(mapObject, teams, loadContext.AssetStore, gameObjects, position);
-                                if (gameObject != null)
```

Comment:
It doesn't make sense to keep the gameObject variable or does it?

Reply:
Right...I'll remove it.